### PR TITLE
feat: fix upload component files show

### PR DESCRIPTION
**Pull request by o0OooO**
## Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

File: javascript/txt2img-config.js [hunk 1]
```diff
@@ -37,10 +37,23 @@ window.onload = function() {
 let uploadedFilesMap = new Map();
 let chunkSize = 512 * 1024 * 1024; // 200MB chunk size, you can adjust this as needed.
 
+function clearFileInput() {
+    var fileInput = document.getElementById('file-uploader');
+    var newFileInput = document.createElement('input');
+    newFileInput.type = 'file';
+    newFileInput.id = 'file-uploader';
+    newFileInput.className = 'lg secondary gradio-button svelte-1ipelgc';
```
Comment (yike5460) [inline javascript/txt2img-config.js:45]: is svelte-1ipelgc fixed?

**Review by yike5460 — COMMENTED** [review]
please check if the element id won't change in the future, or make it explicit in a marco for future update

File: javascript/txt2img-config.js [hunk 2]
```diff
@@ -236,37 +260,79 @@ function uploadFileChunks(file, presignedUrls, checkpointId, groupName, url, api
...
+              reject();
+            };
+
+            xhr.upload.onprogress = function (event) {
+              const percentComplete = (event.loaded / event.total) * 100;
+              // console.log(`Upload progress: ${percentComplete.toFixed(2)}%`);
+              updatePercentProgress(`${percentComplete.toFixed(2)}%`);
+            };
+            xhr.send(chunk);
+            // fetch(presignedUrls[currentChunk], {
```
Comment (yike5460) [inline javascript/txt2img-config.js:305]: remove unused code

File: javascript/txt2img-config.js [hunk 3]
```diff
@@ -37,10 +37,23 @@ window.onload = function() {
...
+    newFileInput.onchange = showFileName;
+    fileInput.parentNode.replaceChild(newFileInput, fileInput);
+}
+
 function getModelTypeValue(dropdowm_value){
     const typeDom = document.getElementById("model_type_value_ele_id");
     typeDom.value = dropdowm_value
-    return dropdowm_value
+    clearFileInput();
+    return dropdowm_value;
```
Comment (yike5460) [inline javascript/txt2img-config.js:56]: type, dropdown

File: javascript/txt2img-config.js (hunk 1 above)
Comment (yike5460) [reply javascript/txt2img-config.js:45]: use MARCO if such string is inevitable for successive maintenance

Comment (o0OooO) [reply javascript/txt2img-config.js:45]: 目前页面的样式都是临时浏览器抓出来找相似的直接复制进去调的 按钮的样式属性设置的很临时 后面咱是不是可以有前端同学专业统一调整支持下

File: middleware_api/lambda/model_and_train/checkpoint_api.py [hunk 4]
```diff
@@ -150,14 +150,19 @@ class UpdateCheckPointEvent:
 # PUT /checkpoint
 def update_checkpoint_api(raw_event, context):
     event = UpdateCheckPointEvent(**raw_event)
-
+    headers = {
+        'Access-Control-Allow-Headers': 'Content-Type',
+        'Access-Control-Allow-Origin': '*',
+        'Access-Control-Allow-Methods': 'OPTIONS,POST,GET'
+    }
```
Comment (alvindaiyan) [inline middleware_api/lambda/model_and_train/checkpoint_api.py:157]: the lambda infra in cdk project need update as well, otherwise this will not working

Comment (alvindaiyan) [reply middleware_api/lambda/model_and_train/checkpoint_api.py:157]: refer to file infrastructure/src/sd-train/chekpoint-create-api.ts#L117-142

Comment (o0OooO) [reply middleware_api/lambda/model_and_train/checkpoint_api.py:157]: 这个ts里面不是已经处理过了么

Comment (o0OooO) [reply middleware_api/lambda/model_and_train/checkpoint_api.py:157]: 了解了，checkpoint-update-api.ts已经调整

**Review by alvindaiyan — APPROVED** [review]
goooood to me

**Pull request merged**